src/structure/endpoint.rs

```diff
@@ -27,3 +29,18 @@ impl Entity for Endpoint {
         self.guid
     }
 }
+
+/// Used to indicate whether the [Endpoint] supports instance lifecycle
```

Comment:
backticks around `endpoint`

Reply:
my rule of thumb is "when clippy::pedantic complains"

any references to objects in code should use backticks, which formats the object like `this`.
square brackets creates a link to the documentation for that object. so internal links should use both backticks and square brackets